planner/core/planbuilder.go

```diff
@@ -796,9 +796,11 @@ func buildShowNextRowID() *expression.Schema {
 func buildShowDDLFields() *expression.Schema {
 	schema := expression.NewSchema(make([]*expression.Column, 0, 4)...)
 	schema.Append(buildColumn("", "SCHEMA_VER", mysql.TypeLonglong, 4))
-	schema.Append(buildColumn("", "OWNER", mysql.TypeVarchar, 64))
+	schema.Append(buildColumn("", "OWNER_ID", mysql.TypeVarchar, 64))
+	schema.Append(buildColumn("", "OWNER_ADDRESS", mysql.TypeVarchar, 20))
 	schema.Append(buildColumn("", "RUNNING_JOBS", mysql.TypeVarchar, 256))
 	schema.Append(buildColumn("", "SELF_ID", mysql.TypeVarchar, 64))
+	schema.Append(buildColumn("", "ORIGINAL_QUERY", mysql.TypeVarchar, 256))
```

Comment:
s/ORIGINAL_QUERY/QUERY/?

Reply:
I am sorry that I can not understand what do you mean.
I was wondering if you could make it more clear?

Reply:
Would you mean replace the "ORIGINAL_QUERY" to "QUERY" ?

Reply:
Yeah.

Reply:
Great.
QUERY is more clear than ORIGINAL_QUERY.
ORIGINAL is confusing.